containers/ecr-viewer/src/app/tests/utils.test.tsx

```diff
@@ -64,6 +64,28 @@ describe("Utils", () => {
 
       expect(actual.availableData[0].value).toEqual("Other");
     });
+    it("should return relgion if available", () => {
```

Comment:
🔬I wouldn't have even bothered if this was the only thing needing changed but you may as well fix it when you're reverting the Dockerfile changes 
```suggestion
    it("should return religion if available", () => {
```